src/xcache-consistency-check

```diff
@@ -559,15 +627,28 @@ def set_configuration():
     config = ConfigParser.ConfigParser(args)
     config_file = cmdline_args.config_file
     if(config_file):
-        log.info("configuration file: "+config_file)
+        print("configuration file: "+config_file)
         config.read(config_file)
-        args['path']                   = config.get('Main', 'path')
-        args['db']                     = config.get('Main', 'db')
-        args['num_procs']              = config.getint('Main', 'num_procs')
-        args['last_check_threshold']   = config.getint('Main', 'last_check_threshold')
-        args['logfile']                = config.get('Main', 'logfile')
+        args['path']                   = config.get('Main', 'path', args)
+        args['db']                     = config.get('Main', 'db', args)
+        args['logfile']                = config.get('Main', 'logfile', args)
+
+        # Only 'config.get' allows to provide a default, 'getint' and 'getboolean' don't
+        # so let's prevent crashing because some attributes aren't defined in the config file
+        try:
+            args['num_procs']              = config.getint('Main', 'num_procs')
+        except:
+            pass
```

Comment:
The problem with doing it this way is that if somebody _does_ specify `num_procs` but gets it wrong, it will just silently use the default.